### ISSUE TITLE
Login Button should not be disabled

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#4483]
On login page, you should be able to click the login button without the form being prefilled. The request should instead be stopped and had messages shown by form validators.

**Comment by Zeyber** [comment]
QA Checks (feature-level < 1.3): 
- [ ] Login button should be disabled when form is invalid and behave normally when valid.
QA Checks (feature-level = 1.3):
- [ ] Login button should be enabled when form is invalid or valid state.
- [ ] Login button click should not submit but show field validators when form is invalid.
- [ ] Login button click should submit when form is valid.

**Comment by KateChuen** [comment]
- Tested locally. 

- [x] feature-level < 1.3: Login button should be disabled when form is invalid and behave normally when valid.

feature-level = 1.3: 
- [x] Login button should be enabled when form is invalid or valid state.
- [x]  Login button click should not submit but show field validators when form is invalid.
- [x]  Login button click should submit when form is valid.
